web3/providers/eth_tester/main.py

```diff
@@ -55,7 +81,7 @@ def __init__(self, ethereum_tester=None, api_endpoints=None):
         else:
             self.api_endpoints = api_endpoints
 
-    def make_request(self, method, params):
+    def make_request(self, method: RPCEndpoint, params: Any) -> RPCResponse:
```

Comment:
At minimum do we know that params is a `List[Any]`?  I'm not 100% sure but I think that we have some knowledge that params is a list of values.

Reply:
> ``params``  This will be a list or other iterable of the parameters for the JSON-RPC method being called.

[Yup!](https://github.com/ethereum/web3.py/blob/0617444b243d878708997009869fe5fd67a57041/docs/internals.rst#writing-your-own-provider), or at minimum an `Iterable`?

Reply:
Making this update touches a lot of files, so to avoid from bloating this pr, i'll follow it up with another that tightens this up